docs/quickstart/quickstart_chains/arbitrum.md

```diff
@@ -0,0 +1,305 @@
+# Arbitrum Quick Start
+
+## Goals
+
+The goal of this quick start guide is to index the total claimed dividends paid to users on the WINR staking contract on Arbitrum.
+
+::: warning
+Before we begin, **make sure that you have initialised your project** using the provided steps in the [Start Here](../quickstart.md) section. 
+:::
```

Comment:
The format of this warning box is quite different from other quick starts. You should align all.

I quite like that we say "Please initialise a Arbitrum Nova Project"

Reply:
Added.